interview-questions.md

```diff
@@ -8,36 +8,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: 
+To add the foreign key we simply need to add a new column in the terminal and run a migrate so it will populate in the scema. The foreign key when added will give a number value that can be associated with a specific student. The foreign key would belong to cohort. This is because the cohort model needs to have something to associate with students, so when we add a student to the cohort we can give student the foreign key and it will belong_to the main model which is Cohort. 
```

Comment:
The key to remember here is the foreign key always goes on the belongs_to model.  In this case, a cohort has many students, and a student belongs_to a cohort, so the cohort_id column would be added to the students table via a migration.  You had the idea right, just switched the models.  😊